class/defaults.yml

```diff
@@ -48,6 +52,20 @@ parameters:
           cpu: 200m
           memory: 200Mi
 
+    slos:
+      enabled: true
+      namespace: appcat-slos
```

Comment:
we should probably need to add `syn-` so that alerts work. I had to do the same for the appcat api server.

Reply:
Alerts will work regardless. But yes we'll need to change the namespace on APPUiO Cloud so that we don't alert Aldebaran. But I thought I'd be consistent with the other namespaces in the component defaults